### PR TITLE
Added logout function and navigation from header

### PR DESCRIPTION
**Pull request by linda-94-andersson**
It's now possible to log out and in through the header and navigate to home.

### CODE REVIEW

File: src/components/Header/Header.tsx [hunk 1]
```diff
@@ -1,15 +1,38 @@
...
+
+    const handleLogo = () => {
+        navigate('/');
+    };
+
+    const handleLogIn = () => {
+        navigate('/login');
+    };
+
+    const handlelogOut = () => {
```
Comment (Zerowhizkey) [inline src/components/Header/Header.tsx:17]: handlelogOut = handleLogOut ??? looks amazing tho !

**Review by Zerowhizkey — CHANGES_REQUESTED** [review]
Loooks awesome, fix the req then u are good to goooooo !!!

File: src/components/Header/Header.tsx (hunk 1 above)
Comment (linda-94-andersson) [reply src/components/Header/Header.tsx]: It's updated now! :)

**Review by Zerowhizkey — APPROVED** [review]
Niiiice !!!

**Pull request merged**